cookbooks/swift/recipes/configs.rb

```diff
@@ -134,6 +138,19 @@
     end
   end
   (1..node['nodes']).each do |i|
+    bind_ip = "127.0.0.1"
+    bind_port = "60#{i}#{p}"
+    if p == 0 && node['servers_per_port'] > 0 then
```

Comment:
p == 0 is a correct, but non-obvious way to spell "this is the object service", maybe:

    if service == "object" && ... then
      ...